charts/newrelic-infrastructure/values-windows.yaml

```diff
@@ -0,0 +1,16 @@
+# Reference yaml for Windows V3 integration
+
+global:
+  licenseKey: <> 
+  cluster: <> 
+
+kubelet:
+  enableWindows: true
```

Comment:
Should we upstream `enableWindows` and `windowsOsList` in the chart's values.yaml file?

Reply:
Updated